crates/db_views/src/combined/modlog_combined_view.rs

```diff
@@ -78,101 +88,117 @@ impl ModlogCombinedViewInternal {
-      .or(mod_transfer_community::other_person_id.eq(other_person));
-
-    let comment_join = mod_remove_comment::comment_id.eq(comment::id);
-
-    let post_join = admin_purge_comment::post_id
-      .eq(post::id)
-      .or(mod_feature_post::post_id.eq(post::id))
-      .or(mod_lock_post::post_id.eq(post::id))
-      .or(
-        mod_remove_comment::id
-          .is_not_null()
-          .and(comment::post_id.eq(post::id)),
-      )
-      .or(mod_remove_post::post_id.eq(post::id));
-
-    let community_join = admin_purge_post::community_id
-      .eq(community::id)
-      .or(mod_add_community::community_id.eq(community::id))
-      .or(mod_ban_from_community::community_id.eq(community::id))
-      .or(
-        mod_feature_post::id
-          .is_not_null()
-          .and(post::community_id.eq(community::id)),
-      )
-      .or(mod_hide_community::community_id.eq(community::id))
-      .or(
-        mod_lock_post::id
-          .is_not_null()
-          .and(post::community_id.eq(community::id)),
-      )
-      .or(
-        mod_remove_comment::id
-          .is_not_null()
-          .and(post::community_id.eq(community::id)),
-      )
-      .or(mod_remove_community::community_id.eq(community::id))
-      .or(
-        mod_remove_post::id
-          .is_not_null()
-          .and(post::community_id.eq(community::id)),
-      )
-      .or(mod_transfer_community::community_id.eq(community::id));
-
-    let instance_join = admin_allow_instance::instance_id
-      .eq(instance::id)
-      .or(admin_block_instance::instance_id.eq(instance::id));
+    let community_join = community::table.on(
+      admin_purge_post::community_id
+        .eq(community::id)
+        .or(mod_add_community::community_id.eq(community::id))
+        .or(mod_ban_from_community::community_id.eq(community::id))
+        .or(
+          mod_feature_post::id
+            .is_not_null()
+            .and(post::community_id.eq(community::id)),
+        )
+        .or(mod_hide_community::community_id.eq(community::id))
+        .or(
+          mod_lock_post::id
+            .is_not_null()
+            .and(post::community_id.eq(community::id)),
+        )
+        .or(
+          mod_remove_comment::id
+            .is_not_null()
+            .and(post::community_id.eq(community::id)),
+        )
+        .or(mod_remove_community::community_id.eq(community::id))
+        .or(
+          mod_remove_post::id
+            .is_not_null()
+            .and(post::community_id.eq(community::id)),
+        )
+        .or(mod_transfer_community::community_id.eq(community::id)),
+    );
+
+    let instance_join = instance::table.on(
+      admin_allow_instance::instance_id
+        .eq(instance::id)
+        .or(admin_block_instance::instance_id.eq(instance::id)),
+    );
+
+    let community_actions_join = community_actions::table.on(
+      community_actions::community_id
+        .eq(community::id)
+        .and(community_actions::person_id.nullable().eq(my_person_id)),
```

Comment:
This is the only new join, needed to get your moderator status for listing type filtering.